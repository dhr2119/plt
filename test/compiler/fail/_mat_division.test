int main() {
  matrix int[2,2] m1;
  matrix int[2,2] m2;  
  m1 = [1,2;3,4];
  m2 = [5,6;7,8];
  m2 = m2 / m1;
  return 0;
}
