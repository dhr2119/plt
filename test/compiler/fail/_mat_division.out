Fatal error: exception Exceptions.UnsupportedMatrixBinop("Cannot divide matrices")
